lib/Basics/Exceptions.h

```diff
@@ -58,6 +58,25 @@
 #define THROW_ARANGO_EXCEPTION_MESSAGE(code, message) \
   throw arangodb::basics::Exception(code, message, __FILE__, __LINE__)
 
+/// @brief throws an arango result if the result fails
+#define THROW_ARANGO_EXCEPTION_IF_FAIL(reSUlt) \
```

Comment:
why `reSUlt` and not `result`?

Reply:
There was a problem with the macro expansion that could be fixed by slightly different spelling of the parameter.